libs/shared/src/components/CarrierBox.vue

```diff
@@ -18,13 +18,14 @@
 
 <script lang="ts" setup>
 import {toRefs} from 'vue';
-import {type CarrierName} from '@myparcel/constants';
+import {resolveCarrierName} from '../utils';
+import {type CarrierIdentifier} from '../types';
```

Comment:
Ik denk dat je deze import beter vanuit `@myparcel-do/shared` kunt halen zoals je al doet in `PickupLocationCarrierGroup.vue`

Reply:
Thanks, ik heb het gewijzigd.

Reply:
Ik heb het gewijzigd